pkg/cluster/cluster.go

```diff
@@ -80,7 +80,6 @@ type manager struct {
 	armPublicIPAddresses     armnetwork.PublicIPAddressesClient
 	loadBalancers            network.LoadBalancersClient // TODO: use armLoadBalancers instead.
 	armLoadBalancers         armnetwork.LoadBalancersClient
-	privateEndpoints         network.PrivateEndpointsClient // TODO: use armPrivateEndpoints instead.
```

Comment:
Can we remove /pkg/util/azureclient/mgmt/network/privateendpoints.go ?

Seems it was only referenced here.

Reply:
Not yet. `fpPrivateEndpoints` still uses the old one.